### ISSUE TITLE
manifest() throws error in Node.js 0.12.0

### ISSUE THREAD

**Issue by SAnDAnGE** [sindresorhus/gulp-rev#91]
```javascript
gulp.task('jsext', function() {
    return gulp.src(config.js.ext.src, {base : config.pubdevDir})
        .pipe($.plumber())
        .pipe($.concat('resources/js/ext.js'))
        .pipe($.rev())
        .pipe(gulp.dest(config.publicDir))
        .pipe($.rev.manifest(/*{path: 'jsext-manifest.json'}*/));
        // .pipe(gulp.dest(config.manifestsPath));
});
```
I commented the line above just to see where the error is generated.. and it seems somewhere in the manifest file:
```winbatch
C:\Zend\Apache2\htdocs\u3c>gulp jsext
[00:39:40] Using gulpfile C:\Zend\Apache2\htdocs\u3c\gulpfile.js
[00:39:40] Starting 'jsext'...
[00:39:40] 'jsext' errored after 129 ms
[00:39:40] Error: ENOENT, stat 'C:\Zend\Apache2\htdocs\u3c\rev-manifest.json'
    at Error (native)
```

**Comment by bakape** [comment]
Can also confirm issue. Probably related to https://github.com/gulpjs/gulp/issues/873

**Comment by undoZen** [comment]
it's solved by #84, @sindresorhus plz publish to npm

**Comment by sindresorhus** [comment]
Fixed in 3.0.1
